work_queue/test/TR_work_queue_remotetask.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+set -e
+
+. ../../dttools/test/test_runner_common.sh
+
+import_config_val CCTOOLS_PYTHON_TEST_EXEC
+import_config_val CCTOOLS_PYTHON_TEST_DIR
+PONCHO_PACKAGE_SERVERIZE=$(cd "$(dirname "$0")/../../poncho/src/"; pwd)/poncho_package_serverize
+
+export PYTHONPATH=$(pwd)/../src/bindings/${CCTOOLS_PYTHON_TEST_DIR}:$PYTHONPATH
+
+STATUS_FILE=wq.status
+PORT_FILE=wq.port
+
+run_wq_worker_coprocess()
```

Comment:
I think you should be able to modify the regular run wq worker by setiing  --coprocess 0 as the default. In this way, if we ever have more than one test using the coprocess we don't need to duplicate run_wq_worker_coprocess